.travis.yml

```diff
@@ -6,6 +6,9 @@ python:
 env:
 - DJANGO_VERSION=1.7
 - DJANGO_VERSION=1.8
+- DJANGO_VERSION=1.9
```

Comment:
Do we need to maintain support for EoL versions? `1.8` and `1.10` may be worthwhile as they're still supported for the next year, but `1.7` and `1.9` are both unsupported and considered _insecure_.